conan/tools/cmake/cmakedeps.py

```diff
@@ -552,26 +553,32 @@ def get_name(self, req):
         ret = req.new_cpp_info.get_property("cmake_target_name", self.name)
         if not ret:
             # The old cpp info
-            ret = req.cpp_info.get_name(self.name)
+            ret = req.cpp_info.get_name(self.name, default_name=False)
```

Comment:
This is for packages that have declared ``self.cpp_info.names["CMakeDeps"]``, isn't it? If this is the case, maybe we want to already deprecate it? ConanCenter is not using it at all, I don't think users have started doing this much either. So we can go with:
- New ``cmake_target_name`` property
- Or fallback to ``get_name("cmake_find_package_multi", default_name=False)`` for compatibility with ConanCenter and other existing recipes.